posix.mak

```diff
@@ -65,14 +63,14 @@ PHOBOS_STABLE_FILES_GENERATED := $(subst $(PHOBOS_STABLE_DIR), $(PHOBOS_STABLE_D
 ################################################################################
 
 # stable dub and dmd versions used to build dpl-docs
-DUB_VER=1.1.0
 STABLE_DMD_VER=2.072.2
 STABLE_DMD_ROOT=$(GENERATED)/stable_dmd-$(STABLE_DMD_VER)
 STABLE_DMD_URL=http://downloads.dlang.org/releases/2.x/$(STABLE_DMD_VER)/dmd.$(STABLE_DMD_VER).$(OS).zip
-STABLE_DMD=$(STABLE_DMD_ROOT)/dmd2/$(OS)/$(if $(filter $(OS),osx),bin,bin$(MODEL))/dmd
+STABLE_DMD_BIN_ROOT=$(STABLE_DMD_ROOT)/dmd2/$(OS)/$(if $(filter $(OS),osx),bin,bin$(MODEL))
+STABLE_DMD=$(STABLE_DMD_BIN_ROOT)/dmd
 STABLE_DMD_CONF=$(STABLE_DMD).conf
-STABLE_RDMD=$(STABLE_DMD_ROOT)/dmd2/$(OS)/$(if $(filter $(OS),osx),bin,bin$(MODEL))/rdmd \
-	--compiler=$(STABLE_DMD) -conf=$(STABLE_DMD_CONF)
+STABLE_RDMD=$(STABLE_DMD_BIN_ROOT)/rdmd --compiler=$(STABLE_DMD) -conf=$(STABLE_DMD_CONF)
+DUB=$(STABLE_DMD_BIN_ROOT)/dub
```

Comment:
`DUB` was the existing name, but for consistency it could be renamed to `DUB_STABLE`